include/psa/crypto_config.h

```diff
@@ -86,6 +86,10 @@
 #define PSA_WANT_ALG_SHA_256                    1
 #define PSA_WANT_ALG_SHA_384                    1
 #define PSA_WANT_ALG_SHA_512                    1
+#define PSA_WANT_ALG_SHA3_224                   1
```

Comment:
HMAC-SHA3-224 and HMAC-SHA3-256 are failing systematically. You need to update the definition of `PSA_HMAC_MAX_HASH_BLOCK_SIZE` in `include/psa/crypto_sizes.h`. Also `PSA_HASH_MAX_SIZE` doesn't take SHA3 into account.